pkg/roachprod/vm/ibm/startup.go

```diff
@@ -0,0 +1,154 @@
+// Copyright 2025 The Cockroach Authors.
+//
+// Use of this software is governed by the CockroachDB Software License
+// included in the /LICENSE file.
+
+package ibm
+
+import (
+	"bytes"
+
+	"github.com/cockroachdb/cockroach/pkg/roachprod/vm"
+)
+
+// Startup script used to find/format/mount all local or attached disks.
+// Each disk is mounted as /data<disknum>, and, in addition, a symlink
+// created from /mnt/data<disknum> to the mount point.
+
+// startupArgs specifies template arguments for the setup template.
+type startupArgs struct {
+	vm.StartupArgs
+	ExtraMountOpts   string
+	UseMultipleDisks bool
+}
+
+const startupTemplate = `#!/usr/bin/env bash
+
+# Script for setting up an IBM machine for roachprod use.
+
+# TODO(golgeek): this is required due to a regression in the IBM cloud-init
+# on Ubuntu 22.04 that causes the default user to be locked. This should be
+# removed once the issue is fixed (case CS4275852 is open with IBM) and a new
```

Comment:
I don't seem to see the support case under the shared account. Any indication (on the case) as to when this is planned to be fixed?